openapi/paths/service-info.yaml

```diff
@@ -22,9 +22,14 @@ get:
         ...
         "type": {
             "group": "org.ga4gh",
-            "artifact": "drs"
+            "artifact": "drs",
+            "version": "1.5"
         }
         ...
+        "stats": {
```

Comment:
this does not match what the response sample indicates

additionally, I would prefer some consistency with projects like refget and htsget, where the service-specific additions to the service-info response are nested in an object named after the service artifact

Reply:
Hey David, could you point me to the specifics for refget and htsget. I would love to understand how they are written so we could incorporate those into DRS

Reply:
Thanks for pointing this out @davidlougheed , I think it's a good idea to be consistent here so Michael or I will take a look and try to address your feedback ASAP.

Reply:
For refget, see http://samtools.github.io/hts-specs/refget.html - specifically `GET /sequence/service-info` and the **Example JSON request and response** section below it. They put myriad service-specific service info properties in a `refget` sub-object, which I like as a pattern since it makes it clear it's an extension of the service-info spec.

Reply:
Although I don't feel as strongly about how the stats are nested in the response as much as I feel that they should be nested, in one form or another.